### DISCUSSION TITLE
番茄小说网的正文界面渲染后部分字体无法正常显示

### DISCUSSION THREAD

**Discussion by LinLei0102** [F9y4ng/GreasyFork-Scripts#225] [问答专区(Question / Answer)] [answered]
使用插件后，正文界面的部分字全都变□了，关闭字体重写后就正常了。（实测选择其他字体也有同样问题）
不知道开发者大大能解决不><
![Snipaste_2023-06-11_05-51-51](https://github.com/F9y4ng/GreasyFork-Scripts/assets/48636379/52f0f2fa-66f1-4906-bf53-b4b2790df038)

**Comment by F9y4ng** [comment, marked as answer]
该网站为了防止爬虫爬内容，刻意将部分中文字体做了特殊处理，所以你在使用第三方字体的时候就会出现乱码。

这种情况就字体渲染来说除了停止使用第三方字体外别无他法，除非你能忍受一部分字体是你自定义的，另外一部分字体是网站定义的这种乱78糟的大场面。

**Reply by LinLei0102** [reply]
非常感谢回复>< （狗番茄）
